education/blockchain/hashing.md

```diff
@@ -1,9 +1,4 @@
 # Hashing
-You can think of a hash as a fingerprint. Everything on a computer has its own fingerprint, so if two things have the same hash, you know that they are copies of each other. This is really useful because if you want to make sure everyone's version of the blockchain is correct, you can compare the hash or "fingerprint" to make sure they match.
+A hash is like a fingerprint. Everybody has one, and everyone's is different. If you see a person, it is really easy to make a copy of their fingerprint (just have them press their finger on something). But if you see a fingerprint, it is impossible to create a copy of the person whose fingerprint it is. 
 
-## Specifics
```

Comment:
@kmanc You don't like / think we need an analogy for hashing, or to explain it in further detail?